apps/web/vite.config.ts

```
@@ -0,0 +1,166 @@
+	"react-easy-crop",
+	"tslib",
+];
+
+const rootDir = path.resolve(url.fileURLToPath(import.meta.url), "../../..");
+
+const replaceImportPlugin = (
+	baseDir: string,
+	sources: Partial<Record<string, string>>,
+	ignoreFn: (
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+		options: Parameters<Extract<NonNullable<Plugin["resolveId"]>, Function>>[2],
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)